README.md

```diff
@@ -124,4 +126,12 @@ To start the registry inside the virtualenv that `pipenv` creates:
 FLASK_APP=app.py pipenv run flask run
 ```
 
-Pipenv should read in the local `.env` file and supply those database connection strings to the application, which will be run by the Flask development server.
\ No newline at end of file
+Pipenv should read in the local `.env` file and supply those database connection strings to the application, which will be run by the Flask development server.
+
+You can then navigate to http://localhost:5000/admin in the browser.
+
+### Debugging
+
+If you are served an error message on the admin home screen, you may need to run `npm install` in the root directory of the library_registry repo. If you're running the application locally, you can also try running the same command in the root directory of the registry_admin repo.
+
+The latter command will only work if the circulation-web and circulation repos are linked using `npm link`. To do this, run `npm link` in the registry_admin repo and then `npm link simplified-registry-admin` in the library_registry repo.
```

Comment:
Add that this is in the non-Docker setup.

Reply:
Fixed!